platform/viewer/src/lib/localFileLoaders/fileLoaderService.js

```diff
@@ -53,7 +53,8 @@ class FileLoaderService extends FileLoader {
     return result;
   }
 
-  addFile(file) {
+  async addFile(file) {
+    const cornerstoneWADOImageLoader = await getCornerstoneWADOImageLoader();
```

Comment:
Do these cause a timing issue with the init that takes place in `ViewerRouting`? The assumption is that the init has completed, right?

Reply:
> Do these cause a timing issue with the init that takes place in ViewerRouting? 

No. Looks like this does not get called from `ViewerRouting` at all.

There are call paths to this from `ViewerLocalFileData` and  `StudyListRoute`.
`ViewerLocalFileData` is now being shown only after initialising cornerstoneWADOImageLoader.

Call path from `StudyListRoute` is from unused `onDrop` method.